config/routes.rb

```diff
@@ -0,0 +1,58 @@
+Rails.application.routes.draw do
+  resources :microposts
+  resources :users
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+  root 'application#hello'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.